UT4MasterServer/Controllers/AccountController.cs

```diff
@@ -182,7 +182,7 @@ public async Task<IActionResult> RegisterAccount([FromBody] RegisterRequest requ
 		}
 
 		// TODO: should we also get user's email?
-		await accountService.CreateAccountAsync(request.Username, request.Password); // TODO: this cannot fail?
+		await accountService.CreateAccountAsync(request.Username, request.Email, request.Password); // TODO: this cannot fail?

```

Comment:
There should be some validation of the email on server-side. What checks i would prefer:
- Verify that email is not a duplicate already stored in db
- `Regex` validation to check proper format
- Maybe additionally a check for domains such as `mailinator.com` to prevent people from creating multiple accounts too easily.